lib/dlss/capistrano/tasks/deployed_branch.rake

```diff
@@ -1,5 +1,5 @@
 desc 'display the deployed branch and commit'
-task deployed_branch: ['controlmaster:setup', 'otk:generate'] do
+task :deployed_branch do
```

Comment:
We no longer need to do this in each task file because it's done in `setup/setup.rake`